cmd/keys.go

```diff
@@ -55,6 +56,22 @@ func keysCmd(a *appState) *cobra.Command {
 	return cmd
 }
 
+func keysUseCmd(a *appState) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     "use chain_name key_name",
+		Aliases: []string{"a"},
+		Short:   "Use a key from the keychain associated with a particular chain. Look at ~/.relayer/keys/ibc-0/keyring-test ",
```

Comment:
Nit: I don't think the "Look at ...path..." is necessary. 

My preference would be to either remove that sentence. OR maybe point them to the `rly keys list` command instead.

Reply:
Will do . I will make that update along with the read me file change .